systemd/afterburn-checkin.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Afterburn (Check In)
 ConditionKernelCommandLine=|ignition.platform.id=azure
+After=network.target
```

Comment:
I think it should depend on After=network-online.target instead

Reply:
https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/

Maybe not. If afterburn retries, `network.target` is probably correct.

Reply:
Hmm fair, then LGTM

Reply:
Indeed, this is only to give a change to NM/sd-networkd to start before afterburn.
There is no way to guarantee that the network is stable before running the checkin logic.
Afterburn interally retries (with backoff) to be resilient to that.